ext/tiny_tds/client.c

```diff
@@ -190,7 +196,6 @@ static void rb_tinytds_client_free(void *ptr) {
     cwrap->closed = 1;
     cwrap->userdata->closed = 1;
   }
-  xfree(cwrap->userdata);
```

Comment:
Just realized that this line change might have been left over from a rebase. I'll have to take another look to see if it's needed or not.